app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -112,6 +116,15 @@ class Config {
 
   Future<List<Map<String, dynamic>>> get luciProdBuilders => _getLuciBuilders('prod');
 
+  Future<List<Map<String, dynamic>>> get flutterLuciTryBuilders => _getRepoLuciBuilders('try', 'flutter');
+  Future<List<Map<String, dynamic>>> get engineLuciTryBuilders => _getRepoLuciBuilders('try', 'engine');
+  Future<List<Map<String, dynamic>>> get cocoonLuciTryBuilders => _getRepoLuciBuilders('try', 'cocoon');
+  Future<List<Map<String, dynamic>>> get packagesLuciTryBuilders => _getRepoLuciBuilders('try', 'pacakges');
+  Future<List<Map<String, dynamic>>> get flutterLuciProdBuilders => _getRepoLuciBuilders('prod', 'flutter');
+  Future<List<Map<String, dynamic>>> get engineLuciProdBuilders => _getRepoLuciBuilders('prod', 'engine');
+  Future<List<Map<String, dynamic>>> get cocoonLuciProdBuilders => _getRepoLuciBuilders('prod', 'cocoon');
+  Future<List<Map<String, dynamic>>> get packagesLuciProdBuilders => _getRepoLuciBuilders('prod', 'packages');
```

Comment:
Wouldn't  be easier to expose _getRepoLuciBuilders instead? using variables you will need a switch case of many ifs to get the correct values.

Reply:
Yeah, that would be easier, but a couple of APIs need these values for tests..

Reply:
Oh, wait. Let me try something.

Reply:
I don't think keeping variables because are referenced in tests is a valid reason to keep them. I suggest removing and fixing tests if required.

Reply:
You are right. I have removed them in the recent commit. You may need to refresh to see the changes.